recipe/meta.yaml

```diff
@@ -19,32 +17,44 @@ build:
   skip: true  # [win and vc<14]
   missing_dso_whitelist:  # [linux and s390x]
     - $RPATH/ld64.so.1    # [linux and s390x]
+  script_env:
+    # "Environment variable VCPKG_FORCE_SYSTEM_BINARIES must be set on
+    # arm, s390x, ppc64le and riscv platforms."
+    - VCPKG_FORCE_SYSTEM_BINARIES=1  # [ arm64 or aarch64 or s390x or ppc64le]
+  overlinking_ignore_patterns:
+    # complains about libfmt.so
```

Comment:
We should investigate. If it complains, it's usually for good reasons.

Reply:
I guess that's what the patch was for, but it failed. I will give it a try.

Reply:
`spdlog` relies on `fmt`, see https://metayaml-conda.fly.dev/metayaml/reverse_dependencies?name=fmt

Reply:
Turned out tiledb needs fmt, and for other platforms it was added implicitly thru spdlog, I've fixed it.